diff --git a/platform.h b/platform.h
index dc875d0..d720876 100644
--- a/platform.h
+++ b/platform.h
@@ -58,6 +58,7 @@ namespace ftgl {
     char * strndup( const char *s1, size_t n);
 #elif defined(_WIN32) || defined(_WIN64)
     /* does not exist on windows */
+    #define strdup _strdup
     char * strndup( const char *s1, size_t n);
 #	if !defined(_MSC_VER) || _MSC_VER < 1800
 		double round(double v);
